internal/test/wmcb/wmcb_test.go

```diff
@@ -290,6 +489,81 @@ func handleCSRs() error {
 	return nil
 }
 
+// createCNIConf create the local cni.conf and returns it's path
+func createCNIConf(ovnHostSubnet string) (string, error) {
```

Comment:
nit: This function can be broken down further to update network CR and generate template file.

Reply:
I don't follow. I am not updating the network CR here.

Reply:
Ohh, sorry, I misread the function. `getServiceCIDR` and `generateCNIFile`